core/commonTest/src/ErrorTests.kt

```diff
@@ -13,6 +13,7 @@ class ErrorTests {
     @Test
     fun testExtraArguments() {
         val argParser = ArgParser("testParser")
+        argParser.outputAndTerminate = { message, _ ->  error(message) }
```

Comment:
Can be extracted into a helper method in `ErrorTests` or beside: 
`fun ArgParser.avoidProcessExit() = apply { outputAndTerminate = ... }`